sdk/go/.changes/v0.13.0.md

```diff
@@ -0,0 +1,19 @@
+## sdk/go/v0.13.0 - 2024-09-11
+
+This SDK uses 🚙 Engine + 🚗 CLI version `v0.13.0`. [See what changed in that release](https://github.com/dagger/dagger/releases/tag/v0.13.0).
+
+🐹 https://pkg.go.dev/dagger.io/dagger@v0.13.0
+
+
+### 🔥 Breaking Changes
+- Fix optional function arguments being generated as required in codegen by @helderco in https://github.com/dagger/dagger/pull/8106 \
+  If there's a Module dependency with a non-null function argument, it will
+  change from required to optional, in the generated client bindings.
```

Comment:
Ooops... missed the "default value" part:
```suggestion
  If there's a Module dependency that has a non-null Function argument with a default value,
  it will change from required to optional, in the generated client bindings.
```